Mage.Common/src/main/java/mage/view/CardView.java

```diff
@@ -287,6 +293,7 @@ private static String getCardTypeLine(Game game, Card card) {
      */
     public CardView(Card card, Game game, boolean controlled, boolean showFaceDownCard, boolean storeZone) {
         super(card.getId(), card.getExpansionSetCode(), card.getCardNumber(), card.getUsesVariousArt(), game != null);
+        this.setOriginalValues(card);
         this.originalObject = card;
```

Comment:
`originalObject` field can be removed from the class now?

Reply:
Not, as stated above, there are still 7 places where `getOriginalCard` is called.
I'm digging what the card rating is doing, and if there is enough info in a given CardView for it.

Reply:
* CSV card rating used for AI bot drafting and for cards viewer in GUI (just for info).
* `PermanentEvaluator` and other things used in AI game simulations to calculate battlefield score and choose best action for AI.

P.S. I can see another card object usage in the views like `PermanentView` (example: second side info/transformation). Maybe it can be replaced by CardView too.

Reply:
Ok I have cleaned both. Moved RateCard to mage-common to extend it to CardView arguments. Almost identical except that the CardView mode can not determine that a card is a removal by looping through abilities/effects.
Tested the card renderer debug tool and it works well with dfc.